CardoLibs/IO/TextFile/TextFileWriter/TextFileWriter.py

```diff
@@ -0,0 +1,12 @@
+from CardoExecutor.Common.CardoContext import CardoContext
+from CardoExecutor.Common.CardoDataFrame import CardoDataFrame
+from CardoExecutor.Contract.IStep import IStep
+
+
+class TextFileWriter(IStep):
```

Comment:
do you really need `TextFileWriter`? why not saving it as CSV?
that way you can keep your column names